src/test-gui/java/de/qabel/desktop/ui/accounting/AccountingGuiTest.java

```diff
@@ -32,7 +32,7 @@ public void testAddsIdentity() throws EntityNotFoundExcepion, PersistenceExcepti
         page.add().inputAndConfirm("a new identity");
 
         Identities identities = identityRepository.findAll();
-        assertEquals(1, identities.getIdentities().size());
+        assertEquals(2, identities.getIdentities().size());
```

Comment:
In Android the tests that need an identity clear all identities and contacts in the setUp method.Together with a hard coded test private key this shouldn't have a performance impact on the tests, esp. if we use in memory databases/repositories.